Yoast/wordpress-seo
# Add store to replacevars plugins and dispatch action to fill store.

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Decided to send each replacevar separately, rather than creating a new action.

## Test instructions

This PR can be tested by following these steps:

* Load the edit-post page in Gutenberg with the console open, you should see many updateReplacevars actions.

## Quality assurance


Partial fix of #9531
